app/src/main/kotlin/com/simplemobiletools/voicerecorder/fragments/PlayerFragment.kt

```diff
@@ -162,10 +167,20 @@ class PlayerFragment(context: Context, attributeSet: AttributeSet) : MyViewPager
     }
 
     private fun getRecordings(): ArrayList<Recording> {
-        return if (isQPlus()) {
-            getMediaStoreRecordings()
-        } else {
-            getLegacyRecordings()
+        return when {
+            isRPlus() -> {
+                ArrayList(getMediaStoreRecordings() + getSAFRecordings()).apply {
```

Comment:
combine them with addAll please, this looks a bit weird :)

Reply:
Something like this?  @tibbi 

```kotlin
private fun getRecordings(): ArrayList<Recording> {
        val recordings = ArrayList<Recording>()
        return when {
            isRPlus() -> {
                recordings.addAll(getMediaStoreRecordings())
                recordings.addAll(getSAFRecordings())
                recordings
            }
            isQPlus() -> {
                recordings.addAll(getMediaStoreRecordings())
                recordings.addAll(getLegacyRecordings())
                recordings
            }
            else -> {
                recordings.addAll(getLegacyRecordings())
                recordings
            }
        }.apply {
            sortByDescending { it.timestamp }
        }
    }

```

Reply:
maybe ye :)